src/angular/components/ciphers.component.ts

```diff
@@ -80,14 +80,15 @@ export class CiphersComponent {
         if (this.searchTimeout != null) {
             clearTimeout(this.searchTimeout);
         }
+        const deletedFilter: (cipher: CipherView) => boolean = (c) => c.isDeleted === this.deleted;
```

Comment:
This is a little verbose to look at. We can let the type be inferred?

Reply:
I did try that initially, perhaps my inexperience with TypeScript (I did google it a bit) but the compiler would not leave it alone, it could infer the `[]` type, but not the singular type version :-(

Reply:
Weird.